docs/design/2022-06-15-phyical plan explain.md

```diff
@@ -0,0 +1,406 @@
+# TiSpark Design Documents
+
+- Author(s): [Author Name](http://github.com/your-github-id), [Co-Author Name](http://github.com/your-github-id), ...
+- Tracking Issue: https://github.com/pingcap/tispark/issues/2385
```

Comment:
Tracking issue is issue to track the whole lifecycle of a feature proposal, including design, impl, doc, you could refer https://github.com/pingcap/tidb/issues/35258 as an example.